go.mod

```diff
@@ -76,91 +76,12 @@ require (
 )
 
 require (
-	cloud.google.com/go v0.54.0 // indirect
 	github.com/Azure/go-ansiterm v0.0.0-20210617225240-d185dfc1b5a1 // indirect
-	github.com/DataDog/zstd v1.4.8 // indirect
```

Comment:
are all of these dependenices now obsolete? :thinking:

Reply:
i often do `go mod tidy`, so yes :-)